src/icrar/leap-accelerate/math/linear_math_helper.h

```diff
@@ -114,7 +118,7 @@ namespace icrar
         auto v = Eigen::Matrix<T, Eigen::Dynamic, 1>(value.size());
         for(int i = 0; i < value.size(); ++i)
         {
-            v(i) = value(casacore::IPosition(i));
+            v(i) = value(casacore::IPosition(1, i));
```

Comment:
You can use `Array::operator[](size_t)` here

Reply:
Scratch that -- `Array::operator[]` returns `Array<T>`, not `T`.

But maybe you can use `std::copy` instead of having an explicit loop.

Reply:
I'll leave a note here to consider using std::copy however the same situation applies where Eigen::Matrix has its iterators disabled

Reply:
It's a pity that `Eigen::Matrix` cannot be traversed in the normal way, which is something I would have imagined would have been possible. I'm completely unfamiliar with the Eigen API though, so I'm sure there are good reasons.

Reply:
I'll try whats on this page:
https://eigen.tuxfamily.org/dox-devel/group__TutorialSTL.html

apparently I need to specify a 1D view of the matrix to do so.